### ISSUE TITLE
Audio Detection Clearly Shown in Review Page

### ISSUE THREAD

**Issue by Anon666333** [blakeblackshear/frigate#12484]
I want to be able to see clearly (in the review page) when something was detected by audio. Currently this is being marked as "motion" which is not helpful because it gets lost. For reference, I use this to detect my dog barking. If I'm not home, I'd like to see easily when he barked, and then I can review the cameras to see if there was anything that caused him to bark (I.E, Stanger in the yard, knock on the door, whatever) but knowing _when _ he barked is key.

Currently, using HA, I have an automation setup that sends a notice to my phone when a bark is detected. I then look inside Frigate review for the time of the notification, can see the the event and any surrounding events from other cameras.

This works fine, but it would be better if Frigate had an "audio" tab. So it would look like:
Events, Detections, Audio Detections, Motion.

And inside of motion, it would show up as a different colour. Alerts are red, Detections are Orange, motion is Yellow, Audio is green.

Thank you!

**Comment by NickM-27** [comment]
Audio is not showed as motion, it is just not showed at all. We actually had audio in the timeline in early dev versions but it became clear it wasn't really useful because you can't listen to the audio easily without also viewing the recording. To be able to do this without high bandwidth usage would likely require a considerable amount of work saving audio "previews"

**Comment by Anon666333** [comment]
Maybe let me clarify then. Nothing needs to change in terms of the way that the previews are currently shown. Right now, the preview is just like motion, as I would expect. And I agree, it would be hard to "preview" audio without a lot of work.  

I do see a blip in yellow colour in my timeline when audio is detected, so I assume this is categorized as "motion" currently...
This is also why I assumed it was captured as a motion event, my apologies if I was incorrect on this.
![image](https://github.com/user-attachments/assets/7576f0be-5ca5-48ab-b699-3e225ae1939c)

What I'm asking is more-so just to separate it, identify it in some-way that it was highlighted due to audio event. A different colour in the timline, and a section dedicated to "Audio". This doesn't need to preview any audio, it can preview the motion or whatever it's doing now, just to be put into a new section so it's clear that this was captured and highlighted because of audio.

Example of what I'm seeing now:
Sound captured:
![image](https://github.com/user-attachments/assets/b048ac7c-2279-4bbf-9bf0-8f7186fa4a22)


Timeline highlight:
(This is 03:23:00 but I can't show the highlight if I put this over it. As you can see no "motion" was detected but there is a slight highlight for when the audio was detected. But this blends right in with everything else (Alerts, Detections, etc.)
![image](https://github.com/user-attachments/assets/c849dad5-cdb8-47b8-b854-e4345bf45131)

**Comment by NickM-27** [comment]
to be clear the yellow is only motion, audio is not shown here at all. 

> What I'm asking is more-so just to separate it, identify it in some-way that it was highlighted due to audio event. A different colour in the timline, and a section dedicated to "Audio". This doesn't need to preview any audio, it can preview the motion or whatever it's doing now, just to be put into a new section so it's clear that this was captured and highlighted because of audio.

right, as I said it was done that way but we found that it was useless. There was no way to act on the audio data if you did see it because there is no way to listen to it. I wouldn't expect this to be a feature until there is an audio preview or some other solution that allows listening to the audio

**Comment by jmtatsch** [comment]
I also would like to see audio threshold events and detections in the future.

**Comment by Wolk9** [comment]
I think the GUI of the former version was on this behalf much better. Although the total look and feel and the experience is much more professional an profound than the latter. The stability is a great leap forward and the detection is close to perfection, but unfortunately some functionality is not as easy anymore as it seems in comparison to old way. 
I mis for instance the complete way of sending positves/false positives to frigate+. Or did I miss something completely?

**Comment by NickM-27** [comment]
> I mis for instance the complete way of sending positves/false positives to frigate+. Or did I miss something completely?

yes, this exists on the page that is dedicated to the frigate+ page

**Comment by Wolk9** [comment]
> > I mis for instance the complete way of sending positves/false positives to frigate+. Or did I miss something completely?
> 
> yes, this exists on the page that is dedicated to the frigate+ page

that is nice, but never found it. I have an account. I can't find anything about that in the documentation; that is still referring to the old version way.

**Comment by NickM-27** [comment]
It is right in the sidebar 
![image](https://github.com/user-attachments/assets/a4d60ab1-96f0-41ca-9127-55e036757a07)

also worth noting that in 0.15 there is a new details pane for review items on the review page where snapshots can be submitted as well

**Comment by bikemike** [comment]
> Audio is not showed as motion, it is just not showed at all. We actually had audio in the timeline in early dev versions but it became clear it wasn't really useful because you can't listen to the audio easily without also viewing the recording. To be able to do this without high bandwidth usage would likely require a considerable amount of work to implement saving audio "previews"

I think audio level in the timeline would be extremely useful as either an overlay/blend of two semi-transparent colors (one color for video/motion and one color for audio) or a side-by-side. I don't understand how this wouldn't be useful. I can imagine it being really useful to see what may be causing a large audio spike and to hear what it sounds like.
